pkg/services/accesscontrol/eval/common_test.go

```diff
@@ -0,0 +1,16 @@
+package eval
+
+type evaluateTestCase struct {
+	desc        string
+	expected    bool
+	evaluator   Evaluator
+	permissions map[string]map[string]struct{}
```

Comment:
As it took me a bit of time to wrap my head around what we were evaluating against what, I suggest renaming this, wdyt?
```suggestion
	userPermissions map[string]map[string]struct{}
```